### ISSUE TITLE
Feature Request: Make currency format more customisable

### ISSUE THREAD

**Issue by SaifurRahmanMohsin** [responsiv/currency-plugin#32]
Right now, the currency formats are limited to **short** and **long** which are pre-defined. What if I wanted something simple like AUD 500 than $500 AUD? **datetime** field in columns support specifying formats as D-m-Y, etc. If would be nice to have the same conversions here.

**Comment by snipiba** [comment]
exactly... i got nothing only fkna number... without any currency symbol, nothing... this is a guess game for user or what?
